internal/db/hold.go

```diff
@@ -36,43 +36,52 @@ func (h *Hold) FindOne(modelID uuid.UUID, q Matcher) (Record, error) {
 	return nil, ErrNotFound
 }
 
-type MatchIter struct {
-	q  Matcher
-	it *iradix.Iterator
-}
-
-func (mi MatchIter) Next() (Record, bool) {
```

Comment:
Hm. I think it's fine to add the FindMany convenience method, but I'd like to keep the MatchIter method around. I think it'll be useful down the line.